mongo_feature.go

```diff
@@ -70,10 +127,53 @@ func (m *MongoFeature) Close() error {
 }
 
 func (m *MongoFeature) RegisterSteps(ctx *godog.ScenarioContext) {
+	ctx.Step(`^remove all documents from the database`, m.RemoveAllDataFromDatabase)
+	ctx.Step(`^remove all documents in the "([^"]*)" collection`, m.RemoveAllDataFromCollections)
+	ctx.Step(`^remove all documents in the following collections: "([^"]*)"`, m.RemoveAllDataFromCollections)
 	ctx.Step(`^the following document exists in the "([^"]*)" collection:$`, m.TheFollowingDocumentExistsInTheCollection)
 	ctx.Step(`^the document with "([^"]*)" set to "([^"]*)" does not exist in the "([^"]*)" collection$`, m.theDocumentWithSetToDoesNotExistInTheCollection)
 }
 
+func (m *MongoFeature) RemoveAllDataFromDatabase() error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	deletedDocs, err := m.ResetDatabase(ctx, m.Database.Name())
+	if err != nil {
+		return err
+	}
+
+	if deletedDocs.Count == 0 {
```

Comment:
(minor): same, I think you would want to check if the deleteddocs == nil ? otherwise this might panic deletedDocs.Count ?